src/connectors/safe.ts

```diff
@@ -0,0 +1,141 @@
+import { SafeAppProvider } from '@gnosis.pm/safe-apps-provider'
+import SafeAppsSDK, {
+  Opts as SafeOpts,
+  SafeInfo,
+} from '@gnosis.pm/safe-apps-sdk'
+import { getAddress } from 'ethers/lib/utils'
+import { normalizeChainId } from '../utils'
+import { Connector } from './connector'
+import {
+  ConnectorNotFoundError,
+  ProviderNotFoundError,
+  ProviderRpcError,
+} from './errors'
+
+const __IS_SERVER__ = typeof window === 'undefined'
+const __IS_IFRAME__ = !__IS_SERVER__ && window?.parent !== window
```

Comment:
Would be nice to export these flags, so users can use it on their apps to conditionally include the connector or NOT.

Like inside GnosisSafe, the `only` connector you should use if the Safe, the other won't work.

For example in my dapo I'm doing this:

```ts
const isServer = typeof window === 'undefined';
const isIframe = !isServer && window?.parent !== window;

const connectors: any = [
    new MetaMaskConnector({
        appUrl: window.location.href,
    }),
    new WalletConnectConnector({
        qrcode: true,
        rpc: Object.values(supportedChains).reduce((acc, cur) => ({ ...acc, [cur.chainId]: cur.rpcUrls[0] }), {}),
    }),
];

if (isIframe) {
    connectors.push(new SafeConnector({}));
}
```

Maybe is nice if I can import from 'vue-dapp`

```
import {isGnosisSafe} from 'vue-dapp'
```

I believe users shouldn't show other connectors in here, only `GnosisSafe`

<img width="961" alt="Screenshot 2022-10-29 at 12 27 51" src="https://user-images.githubusercontent.com/2920357/198826411-7538a50c-9a7f-411f-9f4d-6ca08ec6746b.png">